velox/core/PlanNode.h

```diff
@@ -1914,13 +1914,16 @@ class UnnestNode : public PlanNode {
   /// must appear in the same order as unnestVariables.
   /// @param ordinalityName Optional name for the ordinality columns. If not
   /// present, ordinality column is not produced.
+  /// @param outer If true, produces a row for each input row even if the
+  /// unnestNames is empty. The output columns are null in this case.
```

Comment:
To make it more readable, one suggested statement:

If true, produces NULL result if array/map input is NULL or empty. It's false by default.